frontend/src/app/modules/landing/landing.component.html

```diff
@@ -0,0 +1,28 @@
+<img class="logo" src="/assets/OpenSpeedMonitor_dark.svg" alt="OpenSpeedMonitor"/>
+
+<main *ngIf="hasData$ | async" [class.center]="showApplicationEmptyState$ | async">
+  <section class="card applications">
+    <h2>{{ 'frontend.de.iteratec.osm.landing.yourApplications' | translate }}</h2>
+    <osm-empty-state *ngIf="showApplicationEmptyState$ | async" image="undraw_fast_loading.svg"
+                     messageKey="frontend.de.iteratec.osm.landing.nothingSetUp"></osm-empty-state>
+
+    <ul class="clickable-list" *ngIf="(applications$ | async)?.length">
+      <li *ngFor="let application of applications$ | async">
+        <a [routerLink]="['/applicationDashboard', application.id]">
+          <aside>
+            <osm-csi-value-small [csiValue]="application.recentCsi?.csiDocComplete" [csiDate]="application.recentCsi?.date"
+                                 [lastResultDate]="application.dateOfLastResults" [showLoading]="application.csiIsLoading">
+            </osm-csi-value-small>
+          </aside>
+          <span class="title">{{application.name}}
+            <span class="detail" *ngIf="application.numPages">{{application.numPages}} Pages</span>
```

Comment:
It seems like the backend doesn't provide information about the number of pages of an application (JobGroupService.groovy:50)

Reply:
That's right. The frontend code here is just a leftover from designing mocked data, as I left out the implementation of "numPages" for a following merge request. I take a look at whether it's easier to add backend logic or clean up frontend logic.

Reply:
I implemented the feature as `pageCount` now.